shopper-s-intention-classification-xgboost-lightgm.py:204: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  pivot2['Month'][k]= datetime.strptime(pivot2['Month'][k], '%b').month
shopper-s-intention-classification-xgboost-lightgm.py:202: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  pivot2['Month'][k] = 6
/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/xgboost/sklearn.py:892: UserWarning: The use of label encoder in XGBClassifier is deprecated and will be removed in a future release. To remove this warning, do the following: 1) Pass option use_label_encoder=False when constructing XGBClassifier object; and 2) Encode your labels (y) as integers starting with 0, i.e. 0, 1, 2, ..., [num_class - 1].
  warnings.warn(label_encoder_deprecation_msg, UserWarning)
../../../input/roshansharma_online-shoppers-intention/online_shoppers_intention.csv











Logistic Regression accuracy is 0.8850

Desicion Tree accuracy is 0.9039

Logistic regression prob auc score: 0.89
Logistic regression auc score: 0.67
[07:43:47] WARNING: ../src/learner.cc:1061: Starting in XGBoost 1.3.0, the default evaluation metric used with the objective 'binary:logistic' was changed from 'error' to 'logloss'. Explicitly set eval_metric if you'd like to restore the old behavior.
XGBoost Classifier accuracy is : 0.8935

              precision    recall  f1-score   support

       False       0.89      0.98      0.94      2599
        True       0.78      0.37      0.50       480

    accuracy                           0.89      3079
   macro avg       0.84      0.67      0.72      3079
weighted avg       0.88      0.89      0.87      3079

              precision    recall  f1-score   support

       False       0.93      0.95      0.94      2599
        True       0.72      0.63      0.67       480

    accuracy                           0.90      3079
   macro avg       0.83      0.79      0.81      3079
weighted avg       0.90      0.90      0.90      3079

              precision    recall  f1-score   support

       False       0.92      0.96      0.94      2599
        True       0.70      0.55      0.62       480

    accuracy                           0.89      3079
   macro avg       0.81      0.76      0.78      3079
weighted avg       0.89      0.89      0.89      3079

Traceback (most recent call last):
  File "shopper-s-intention-classification-xgboost-lightgm.py", line 434, in <module>
    classifier_keras.add(Dense(input_dim = 76,output_dim = 25, init = 'uniform', activation = 'relu' ))
TypeError: __init__() missing 1 required positional argument: 'units'
[REPAIR EXEC TIME]: 9.221518278121948shopper-s-intention-classification-xgboost-lightgm.py:204: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  pivot2['Month'][k]= datetime.strptime(pivot2['Month'][k], '%b').month
shopper-s-intention-classification-xgboost-lightgm.py:202: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  pivot2['Month'][k] = 6
/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/xgboost/sklearn.py:892: UserWarning: The use of label encoder in XGBClassifier is deprecated and will be removed in a future release. To remove this warning, do the following: 1) Pass option use_label_encoder=False when constructing XGBClassifier object; and 2) Encode your labels (y) as integers starting with 0, i.e. 0, 1, 2, ..., [num_class - 1].
  warnings.warn(label_encoder_deprecation_msg, UserWarning)
../../../input/roshansharma_online-shoppers-intention/online_shoppers_intention.csv











Logistic Regression accuracy is 0.8850

Desicion Tree accuracy is 0.9039

Logistic regression prob auc score: 0.89
Logistic regression auc score: 0.67
[06:13:27] WARNING: ../src/learner.cc:1061: Starting in XGBoost 1.3.0, the default evaluation metric used with the objective 'binary:logistic' was changed from 'error' to 'logloss'. Explicitly set eval_metric if you'd like to restore the old behavior.
XGBoost Classifier accuracy is : 0.8935

              precision    recall  f1-score   support

       False       0.89      0.98      0.94      2599
        True       0.78      0.37      0.50       480

    accuracy                           0.89      3079
   macro avg       0.84      0.67      0.72      3079
weighted avg       0.88      0.89      0.87      3079

              precision    recall  f1-score   support

       False       0.93      0.95      0.94      2599
        True       0.72      0.63      0.67       480

    accuracy                           0.90      3079
   macro avg       0.83      0.79      0.81      3079
weighted avg       0.90      0.90      0.90      3079

              precision    recall  f1-score   support

       False       0.92      0.96      0.94      2599
        True       0.70      0.55      0.62       480

    accuracy                           0.89      3079
   macro avg       0.81      0.76      0.78      3079
weighted avg       0.89      0.89      0.89      3079

Traceback (most recent call last):
  File "shopper-s-intention-classification-xgboost-lightgm.py", line 434, in <module>
    classifier_keras.add(Dense(input_dim = 76,output_dim = 25, init = 'uniform', activation = 'relu' ))
TypeError: __init__() missing 1 required positional argument: 'units'
[REPAIR EXEC TIME]: 9.04355788230896shopper-s-intention-classification-xgboost-lightgm.py:204: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  pivot2['Month'][k]= datetime.strptime(pivot2['Month'][k], '%b').month
shopper-s-intention-classification-xgboost-lightgm.py:202: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  pivot2['Month'][k] = 6
/home/cgzhu/anaconda3/envs/migration-plus/lib/python3.6/site-packages/xgboost/sklearn.py:892: UserWarning: The use of label encoder in XGBClassifier is deprecated and will be removed in a future release. To remove this warning, do the following: 1) Pass option use_label_encoder=False when constructing XGBClassifier object; and 2) Encode your labels (y) as integers starting with 0, i.e. 0, 1, 2, ..., [num_class - 1].
  warnings.warn(label_encoder_deprecation_msg, UserWarning)
../../../input/roshansharma_online-shoppers-intention/online_shoppers_intention.csv











Logistic Regression accuracy is 0.8850

Desicion Tree accuracy is 0.9039

Logistic regression prob auc score: 0.89
Logistic regression auc score: 0.67
[14:03:36] WARNING: ../src/learner.cc:1061: Starting in XGBoost 1.3.0, the default evaluation metric used with the objective 'binary:logistic' was changed from 'error' to 'logloss'. Explicitly set eval_metric if you'd like to restore the old behavior.
XGBoost Classifier accuracy is : 0.8935

              precision    recall  f1-score   support

       False       0.89      0.98      0.94      2599
        True       0.78      0.37      0.50       480

    accuracy                           0.89      3079
   macro avg       0.84      0.67      0.72      3079
weighted avg       0.88      0.89      0.87      3079

              precision    recall  f1-score   support

       False       0.93      0.95      0.94      2599
        True       0.72      0.63      0.67       480

    accuracy                           0.90      3079
   macro avg       0.83      0.79      0.81      3079
weighted avg       0.90      0.90      0.90      3079

              precision    recall  f1-score   support

       False       0.92      0.96      0.94      2599
        True       0.70      0.55      0.62       480

    accuracy                           0.89      3079
   macro avg       0.81      0.76      0.78      3079
weighted avg       0.89      0.89      0.89      3079

Traceback (most recent call last):
  File "shopper-s-intention-classification-xgboost-lightgm.py", line 434, in <module>
    classifier_keras.add(Dense(input_dim = 76,output_dim = 25, init = 'uniform', activation = 'relu' ))
TypeError: __init__() missing 1 required positional argument: 'units'
[REPAIR EXEC TIME]: 8.796400785446167